app/src/main/java/acr/browser/lightning/browser/tabs/TabsDesktopAdapter.kt

```diff
@@ -132,7 +132,7 @@ class TabsDesktopAdapter(
             viewHolder.iCardView.isChecked = false
             viewHolder.iCardView.isCheckable = false
             // Background tab
-            TextViewCompat.setTextAppearance(viewHolder.txtTitle, if (tab.isFrozen) R.style.italicText else R.style.normalText)
+            TextViewCompat.setTextAppearance(viewHolder.txtTitle, R.style.italicText)
```

Comment:
This was actually disabling that feature and always using italic for background tabs regardless of whether they are frozen.

Reply:
So this Italic font as it was before didn't work for me, for whatever reason. Therefore the change.

Reply:
Works just fine here. Only frozen tabs are displayed using italic. Loaded background tabs are using normal fonts.